rosa_architecture/about-hcp.adoc

```diff
@@ -188,9 +188,10 @@ endif::openshift-rosa-hcp[]
 ifndef::openshift-rosa-hcp[]
 xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-understand-availability.adoc#rosa-policy-understand-availability[About availability]
 endif::openshift-rosa-hcp[]
-|
+//adding condition to get hcp upgrading PR built
+|ifdef::openshift-rosa-hcp[]
 xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-hcp-upgrading[Upgrading]
-|
+|endif::openshift-rosa-hcp[]
```

Comment:
I think you lose the `|` to fix this? Not sure though.

Reply:
fixed